resources/resources.yaml

```diff
@@ -85,6 +85,13 @@ spec:
               mountPath: /var/log
             - name: logforwarders
               mountPath: /etc/rsyslog.d
+          resources:
+            requests:
+              memory: 100Mi
+              cpu: 100m
+            limits:
+              memory: 200Mi
```

Comment:
Just curious - why would you give a collector less memory than the forwarder and in such a proportion?
Also, I know forwarders had been memory-hungrier, maybe they deserve twice as much until we have sorted them out?

Reply:
In the ticket that I referenced the customer mentions that forwarders are memory hungry so I gave them more. Sure, I'll bump the limits for both.